include/revng/Lift/AdvancedValueInfoPass.h

```diff
@@ -116,7 +116,7 @@ AdvancedValueInfoPass::run(llvm::Function &F,
   auto &SCEV = FAM.getResult<ScalarEvolutionAnalysis>(F);
   AdvancedValueInfo<StaticDataMemoryOracle> AVI(LVI, SCEV, DT, MO, Dispatcher);
 
-#ifndef NDEBUG
+#if NDEBUG
```

Comment:
You sure?

Reply:
mh wait, this was probably something i touched by mistake when i recoved the files.

tho it does not make much sense the original. it was doing extra checks in release only?

Reply:
if *not* def NDEBUG (which means not debug).

Reply:
Uhm, right, make a commit to fix this only.